dev-do-list-server/Controllers/AuthController.cs

```diff
@@ -1,58 +1,82 @@
-﻿using Microsoft.AspNetCore.Http.HttpResults;
+﻿using Azure.Core;
+using DevDoListServer.Jwt;
+using DevDoListServer.Models;
+using DevDoListServer.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.IdentityModel.Tokens;
 using System.IdentityModel.Tokens.Jwt;
+using System.Net;
 using System.Security.Claims;
 using System.Text;
-using System.Net;
-using Newtonsoft.Json;
-using DevDoListServer.Models;
+using System.Text.Json;
 
-namespace DevDoListServer.Jwt
+namespace DevDoListServer.Controllers
 {
-    public static class TokenEndpoint
+    [Route("api/v1/[controller]")]
```

Comment:
Please change [controller] to auth. As it is now, the endpoint will be api/v1/Auth which doesn't match the other controllers